provider-ui/.eslintignore

```diff
@@ -4,5 +4,6 @@ out
 static
```

Comment:
Can you add the cypress command in `makefile` too as done for meshery-ui ?

Reply:
@kushthedude sorry, i could not understand this one, can you clarify?

Reply:
`make run-provider-ui-test` to execute command `cd provider-ui; npm test; cd ..`